common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3270,6 +3270,36 @@
     ],
     "sqlState" : "XXKST"
   },
+  "STV2_ENCODER_UNSUPPORTED_PREFIX_KEY" : {
+    "message" : [
+      "Encoder doesn't support prefix key for state store provider: <stateStoreEncoder>. Please use another state store provider that supports prefix keys, or omit the prefix key."
+    ],
+    "sqlState" : "00000"
```

Comment:
Not sure what sqlState to use here - this is used when we are trying to encode a prefixKey with an encoder that doesn’t support it.